### ISSUE TITLE
[Abandoned Plugin] PrintJobHistory

### ISSUE THREAD

**Issue by jneilliii** [OctoPrint/plugins.octoprint.org#1322]
Yeah I don't have the time to work on it so you all can move ahead with whatever is needed to take this over. same for my PrintJobHistory plugin

_Originally posted by @dojohnso in https://github.com/OctoPrint/plugins.octoprint.org/issues/1319#issuecomment-2332344915_

**Comment by Larry2368** [comment]
I don't have a lot of free time and know very little about GITHUB projects, but I have done some code work in my day.  If someone would like to get in touch via some type of private message, I be interested in thinking about taking these projects over.

Larry

**Comment by KevinVidomski** [comment]
I have 20+ years experience in software development including web technologies.  I have a lot of time on my hands right now.  I'm going to look through the two plugins and see if this is something I could handle.

**Comment by vojtakaniok** [comment]
I created [PR](https://github.com/dojohnso/OctoPrint-PrintJobHistory/pull/31) with fix for one [issue](https://github.com/dojohnso/OctoPrint-PrintJobHistory/issues/23).
Let me know if this plugin is owned by someone once again or I can maintain it for a while.
@jneilliii

**Comment by jneilliii** [comment]
If you'd like to adopt the plugin @vojtakaniok, it's just a matter of submitting a PR similar to the previous adoption PR that dojohnso did [here](https://github.com/OctoPrint/plugins.octoprint.org/pull/1248/files).

**Comment by vojtakaniok** [comment]
Thanks for answer !
I will do some cleanup in the repo and pubslih the PR for adopting.

**Comment by Larry2368** [comment]
I would be willing to help when I can.  My time is often limited.  I am new
to using GitHub and I am not sure what language the plug-in is written in,
but I have been coding since the late '70s.

**Comment by vojtakaniok** [comment]
@Larry2368
Thanks for your willingness to help!  You can checkout my repo and help me to maintain it !
https://github.com/vojtakaniok/OctoPrint-PrintJobHistory

**Comment by jneilliii** [comment]
Thanks @vojtakaniok adoption PR has been merged.

**Comment by Kjubyte** [comment]
@vojtakaniok Do you still actively maintain the project? The disabled issue tracker makes me believe you have abandoned it.

**Comment by vojtakaniok** [comment]
@Kjubyte Yes, I'm still maintaining the project. The issues was disabled by default when forking the repo. 
Fixed
